src/main/webapp/scripts/global-build-stats/chartList.js

```diff
@@ -97,17 +97,62 @@ function ajaxCall(callType, param, successCallback, skipLoading){
 	
 	YAHOO.global.build.stat.wait.modalPopup.render(document.body);
 	if(callType == 'form'){
-		$(param).request(ajaxCallParams);
+		const form = document.getElementById(param);
+		ajaxRequest(form.action, ajaxCallParams, new URLSearchParams(new FormData(form)).toString());
 	} else {
-		new Ajax.Request(param, ajaxCallParams);
+		ajaxRequest(param, ajaxCallParams);

```

Comment:
url seems to be missing here? is that intentional?

Reply:
`param` holds the url. `body` is missing, but it’s not provided to the function - the author calls it ‘link’ mode.

Reply:
But the function just takes a url not an object? Is param actually the url?

Reply:
Yes, the patan is either a url or a form id. See here, for example 
https://github.com/jenkinsci/global-build-stats-plugin/blob/b9e4aaa0bccead0c2196c3ccc9f6009ca3d1433a/src/main/webapp/scripts/global-build-stats/BuildStatConfigs.js#L136